smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/StructureGenerator.kt

```diff
@@ -234,7 +235,7 @@ class StructureGenerator(
     private fun generateErrorStructMembers() {
         if (membersSortedByName.isNotEmpty()) {
             writer.write("")
-            writer.openBlock("public struct Properties {", "}") {
+            writer.openBlock("public struct Properties: \$N {", "}", SwiftTypes.Protocols.Sendable) {
```

Comment:
Modeled errors' internal `Properties` structure (when rendered) is marked `Sendable`.  No other code change required for sendability since all possible model struct properties are already `Sendable`.